hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestIncrementsFromClientSide.java

```diff
@@ -547,4 +553,51 @@ static void assertIncrementKey(Cell key, byte [] row, byte [] family,
   public static String filterStringSoTableNameSafe(final String str) {
     return str.replaceAll("\\[fast\\=(.*)\\]", ".FAST.is.$1");
   }
+
+  /*
+    Test that we have only 1 ttl tag with increment mutation.
+   */
+  @Test
+  public void testIncrementWithTtlTags() throws Exception {
+    LOG.info("Starting " + this.name.getMethodName());
+    final TableName TABLENAME =
```

Comment:
nit: `s\TABLENAME\tableName` since it is not class level static final constant

Reply:
Thank you for the feedback. Addressed in latest commit. Please review again.